_pages/ato/background.md

```diff
@@ -3,12 +3,6 @@ title: Background on the Risk Management Process
 navtitle: Background
 ---
 
-One of the goals of the Federal Information Security Management Act of 2002 (FISMA) is to "provide a comprehensive framework for ensuring the effectiveness of information security controls over information resources that support Federal operations and assets." The National Institute of Standards and Technology (NIST) was tasked with designing and implementing this framework: the result is NIST's Risk Management Framework (RMF). All federal information and information systems (except classified information and national security systems) are subject to NIST's RMF.
-
-There's [an introduction to the RMF on NIST's website](http://csrc.nist.gov/groups/SMA/fisma/framework.html). A more comprehensive guide, including how to apply the framework, references to the various relevant publications, and definitions of roles and responsibilities, is found in NIST's [Special Publication 800-37](http://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-37r1.pdf).
-
-A key role in the RMF is the Authorizing Official (AO). By default this is the agency head, but they can delegate this authority. In the GSA TTS, the Director of Infrastructure is the delegated AO. The AO is ultimately responsible for determining if the risk of operating the system is acceptable, and if so, issuing an Authority to Operate (ATO) for that system.
```

Comment:
AO is defined on the front page now, in more simple words. The hypothesis is that this kind of extra information about FISMA and NIST isn't useful to the majority of the audience. An appendix with links should suffice, and simplifying the reading path to actionable information will pay off.